src/AzureDeprecation.APIs.REST/Contracts/AzureCloud.cs

```diff
@@ -0,0 +1,15 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace AzureDeprecation.APIs.REST.Contracts;
+
+[JsonConverter(typeof(StringEnumConverter))]
+public enum AzureCloud
```

Comment:
I think it's best if we re-use the enums that we already have in the contracts project to avoid duplication

Reply:
Ok, done